packages/replay/src/coreHandlers/handleClick.ts

```diff
@@ -0,0 +1,302 @@
+    if (click && Math.abs(click.timestamp - nowInSeconds()) > 0.01) {
+      click.clickCount++;
+    }
+  }
+
+  /** Try to get an existing click on the given element. */
+  private _getClick(breadcrumb: ClickBreadcrumb): Click | undefined {
+    const { nodeId } = breadcrumb.data;
+    const now = nowInSeconds();
+
+    // Find any click on the same element in the last second
+    // If one exists, we consider this click as a double/triple/etc click
+    return this._clicks.find(click => click.nodeId === nodeId && now - click.timestamp < this._multiClickTimeout);
+  }
+
+  /** Check the clicks that happened. */
+  private _checkClicks(): void {
+    const timedOutClicks: Click[] = [];
+
+    const now = nowInSeconds();
+
+    this._clicks.forEach(click => {
+      if (!click.mutationAfter && this._lastMutation) {
+        click.mutationAfter = click.timestamp <= this._lastMutation ? this._lastMutation - click.timestamp : undefined;
+      }
+      if (!click.scrollAfter && this._lastScroll) {
+        click.scrollAfter = click.timestamp <= this._lastScroll ? this._lastScroll - click.timestamp : undefined;
+      }
+
+      if (click.timestamp + this._timeout <= now) {
+        timedOutClicks.push(click);
+      }
+    });
+
+    // Remove "old" clicks
+    for (const click of timedOutClicks) {
+      this._generateBreadcrumbs(click);
+
+      const pos = this._clicks.indexOf(click);
+      if (pos !== -1) {
+        this._clicks.splice(pos, 1);
+      }
+    }
+
+    // Trigger new check, unless no clicks left
+    if (this._clicks.length) {
+      this._scheduleCheckClicks();
+    }
+  }
+
+  /** Generate matching breadcrumb(s) for the click. */
+  private _generateBreadcrumbs(click: Click): void {
+    const replay = this._replay;
+    const hadScroll = click.scrollAfter && click.scrollAfter <= this._scollTimeout;
+    const hadMutation = click.mutationAfter && click.mutationAfter <= this._threshold;
+
+    const isSlowClick = !hadScroll && !hadMutation;
+    const { clickCount, clickBreadcrumb } = click;
+
+    // Slow click
+    if (isSlowClick) {
+      // If `mutationAfter` is set, it means a mutation happened after the threshold, but before the timeout
+      // If not, it means we just timed out without scroll & mutation
+      const timeAfterClickMs = Math.min(click.mutationAfter || this._timeout, this._timeout) * 1000;
+      const endReason = timeAfterClickMs < this._timeout * 1000 ? 'mutation' : 'timeout';
+
+      const breadcrumb = {
+        message: clickBreadcrumb.message,
+        timestamp: clickBreadcrumb.timestamp,
+        category: 'ui.slowClickDetected',
+        data: {
+          ...clickBreadcrumb.data,
+          url: WINDOW.location.href,
+          route: replay.getCurrentRoute(),
+          timeAfterClickMs,
+          endReason,
+          clickCount,
+        },
+      };
+
+      this._addBreadcrumbEvent(replay, breadcrumb);
+      return;
+    }
+
+    // Rage click
+    if (clickCount > 3) {
```

Comment:
I feel we could consider `3` an option. Not from the get go, lets see how this goes.

Reply:
yes, makes sense! Maybe we need to/want to guard this behind an experiment(al option) anyhow for now 🤔

Reply:
so instead of this arbitrary limit, we now simply capture a new breadcrumb type `ui.multiClick` with a `clickCount`, whenever this is > 1.